operator/pkg/ingress/envoy_config.go

```diff
@@ -360,8 +360,16 @@ func getListenerResource(ingress *slim_networkingv1.Ingress, secretNamespace str
 	}, nil
 }
 
-func getConnectionManager(name string, routeName string) (v2.XDSResource, error) {
+func getConnectionManager(ingress *slim_networkingv1.Ingress, name string, routeName string) (v2.XDSResource, error) {
 	var connectionManager envoy_extensions_filters_network_http_connection_manager_v3.HttpConnectionManager
+	var upgradeConfigs []*envoy_extensions_filters_network_http_connection_manager_v3.HttpConnectionManager_UpgradeConfig
+
+	if annotations.GetAnnotationWebsocketEnabled(ingress) == 1 {
```

Comment:
nit: I reckon enabled/disabled will be widely used with the increase of supported annotations in the future, better to have enums/const for the same. Happy to leave it as what we have now as well.

Reply:
I think I'll leave it as is for now then :)